R/nonmem.R

```diff
@@ -233,28 +242,7 @@ rex::register_shortcuts("babelmixr2")
     identical(expr, quote(`-`)) ||
     identical(expr, quote(`/`))
 }
-#'  Handle d/dt() expression
```

Comment:
I haven't used this, but  doesn't NONMEM support this?

Reply:
I'm not sure. I didn't think so, but maybe.  Does exode2 support it?

Reply:
`rxode2` has supported it for a long time.  I think NONMEM supports it too.

Reply:
Maybe we're talking about a different "this".  Does rxode2 support the `d/dt(compartment)` on the right-hand-side of an assignment?  I thought that it could only be on the left-hand-side of an assignment operator.  (Or, maybe your "this" was something else.)

Reply:
It can be on the right or left side of the equation in `rxode2()`.  Some QSP models use it

Reply:
For now I'm putting it back in, because I think both tools support it.